### DISCUSSION TITLE
Package not showing up in listing

### DISCUSSION THREAD

**Discussion by VelestonRato** [community/community#146102] [npm] [answered]
I have published a package https://www.npmjs.com/package/medusa-plugin-klaviyo and its not showing up in listing. Also i am unable to see the downloads on this package.

**Comment by jamesbrown941** [comment]
"This could be due to an indexing delay on npm's side, which sometimes happens after publishing a package. Try the following:

Ensure the package is published as public (not private).
Double-check your package.json to ensure all metadata (like name, description, and keywords) is correctly filled out.
Wait a few hours, as it might take some time for the package to appear in searches or update download stats.
If the issue persists, reach out to npm support for assistance!"

**Comment by leobalter** [comment, marked as answer]
co-related: https://github.com/orgs/community/discussions/146157#discussioncomment-11451152

Please give it some time for the package to show up and please hold on while we apply a patch to support more from the newly publised packages being featured in the search results.

**Comment by getsettalk** [comment]
i have also published an npm package https://www.npmjs.com/package/rn-build-version

it not showing in search list and auto suggestion, why please fix it.

**Reply by leobalter** [reply]
It may take a few days until the package shows up. I can see it's already there both on the search list and auto suggestion.
